src/store/Textile.ts

```diff
@@ -0,0 +1,11 @@
+import { Context } from "@textile/context";
```

Comment:
In the future, we might want to do a bit more customizing of our access to the Hub APIs, so I have this file here separate so we have a simple place to do that. Note also that I'm using dotenv right now to make it easier to configure the remote endpoint for the Hub API. We can take this out in our "production" app (or configure it).

Reply:
💯